backend/workers/lib/vod_converter/main.py

```diff
@@ -0,0 +1,94 @@
+"""
+Converts between visual object detection dataset formats. See `converter.py` for more info.
+
+To add support for additional data formats, define a module with an `converter.Ingestor` and/or
+`converter.Egestor` implementation and add them to the `INGESTORS` and `EGESTORS` dicts below.
+"""
+
+import argparse
+import logging
+import sys
+
+import lib.vod_converter.converter as converter
+import lib.vod_converter.kitti as kitti
+import lib.vod_converter.kitti_tracking as kitti_tracking
+import lib.vod_converter.coco as coco
+import lib.vod_converter.voc as voc
+import lib.vod_converter.caltech as caltech
+
+logger = logging.getLogger('gunicorn.error')
+logger.setLevel(logging.INFO)
+
+INGESTORS = {
+    'coco': coco.COCOIngestor,
+    'kitti': kitti.KITTIIngestor(),
+    'kitti-tracking': kitti_tracking.KITTITrackingIngestor(),
+    'voc': voc.VOCIngestor(), 
+    'caltech': caltech.CaltechIngestor(), }
+
+
+EGESTORS = {
+    'voc': voc.VOCEgestor(),
+    'kitti': kitti.KITTIEgestor(),
+    'coco': coco.COCOEgestor()
+
+}
+
+
+def main(*, from_path, from_key, to_path, to_key):
+    success, msg = converter.convert(from_path=from_path, ingestor_key=from_key,
+                                     to_path=to_path, egestor_key=to_key,
+                                     select_only_known_labels=False,
+                                     filter_images_without_labels=True, folder_names=None)
+    if success:
+        print(f"Successfully converted from {from_key} to {to_key}.")
+    else:
+        print(f"Failed to convert from {from_key} to {to_key}: {msg}")
```

Comment:
use logger

Reply:
sys.stdout was temporarily redirected to task.info to omit the need for replacing all prints in new scripts